### ISSUE TITLE
Deeplab v3+ I WANT TO TRAIN MY OWN DATASET. TF_INITIALK_CHECKPOINT, INITIALIZE_LAST_LAYER OR EXCLUDE_LIST  

### ISSUE THREAD

**Issue by natha1008** [tensorflow/models#4562]
System information
What is the top-level directory of the model you are using: deeplab
Have I written custom code (as opposed to using a stock example script provided in TensorFlow): Yes
OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux Ubuntu 16.04
TensorFlow installed from (source or binary): binary
TensorFlow version (use command below): 1.6.0
Bazel version (if compiling from source):
CUDA/cuDNN version: 9.0/7.0.4
GPU model and memory: 1080Ti * 2 , 10Gb * 2
Exact command to reproduce:

Describe the problem

This is a feature request. I am trying to train the Deeplab model with my own dataset. I already trained a small set using tf_initial_checkpoint=pascal_train_aug.tar.gz  and initialize_last_layer= False and i got it a MioU accuracy around 60%. However, I need to use my whole dataset that is almost 6K images, and i would like to initialize my new train with the weights from my last train (60%). But, I'm not sure which parameters i have to change.  If, should be like this:
tf_initial_checkpoint = last_checkpoint (60%)
Initialize_last_layer = true 
I also, read some issues with exclude_list={"global_step"}
I don't know if in my case i need to change for exclude_list={"global_step","logits"} 
I'm very confusing with this parameters. Help Pleaseee. 
any idea about my issue

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
